packages/widget/src/components/SendToWallet/SendToWalletButton.tsx

```diff
@@ -44,8 +44,7 @@ export const SendToWalletButton: React.FC<CardProps> = (props) => {
   const hiddenToAddress = hiddenUI?.includes(HiddenUI.ToAddress);
 
   const showInstantly =
-    Boolean(!showSendToWalletDirty && toAddress && !hiddenToAddress) ||
-    requiredToAddress;
+    Boolean(toAddressFieldValue && !hiddenToAddress) || requiredToAddress;
```

Comment:
Some of the values in the Send To Wallet Button (and the Expand Button) where using form values from the config. Switching these to use form values in the state keeps the Send to Wallet button and its open and closed state working as expect